content/guides/template-gds/_index.md

```diff
@@ -0,0 +1,156 @@
+
+   <img src="images/documentation.png" alt="doc" width="550"/>
+<!-- ![Documentation](images/documentation.png) -->
+
+
+## Wokwi Actions
+
+1. **Access Actions Tab**:
+   - Go to the ‘Actions’ tab in your repository to view results. Note that each commit will trigger these actions to run again. Find out more: how the GitHub actions work to build the GDS (link to zero to asic GDS terminology page) files for your project, you can check here (link to /making_asics).
+    
+2. **Actions for Wokwi**:
+
+    - docs: This action generates a preview of how your documentation will look.
+    - fpga:
+    - gds: Action to generate the gds for your project, results for warnings, utilization statistics, and cell usage.
+    - wokwi test:
+    
+
+    <img src="images/actions.png" alt="actionslist" width="300"/>
+
+    Note that red indicates failure and green indicates success. Only the actions 'docs' and 'gds' need to be green
+    Now, the Action results are to be watched to resolve the action failures.
+
+## Troubleshoot Action Failures 
+
+1. **Docs Action**:
+
+   - This action generates a preview of how your documentation will look.
+   - Check the action result to get the status of the check. 
+
+   <img src="images/CheckActions.png" alt="Docs Action Result" width="550"/>
+   
+   - View the Errors and Modify your Docs based on the Error showed, commit the new info.md for actions to rerun. [Tips]() for common cause of errors is shown in the end of this guide.
+
+   <img src="images/checkerrors.png" alt="Docs Action error" width="550"/>
+
+   - Commit will redpo the check and observe the status. 
+
+    <img src="images/changes_docscheck_success.png" alt="Docs Action success" width="550"/>
+
+    - Download the PDF to verify formatting and content.
+
+    <img src="images/actions_pdf.png" alt="Docs Action pdf" width="300"/>
+
+
+<!-- ![Docs Action Result](images/CheckActions.png)
+![Docs Error Result](images/checkerrors.png) -->
+<!--    
+![Docs Success Result](images/changes_docscheck_success.png)
+![Docs Success PDF](images/actions_pdf.png) -->
+
+2. **GDS Action**:
+
+   - This action results in warnings, utilization statistics, and cell usage.
+   - View Action results.
+   - what do the metrics mean - what standard cells are in use? is that what you expected? explain why the results here might not match your drawing
+   - Use the 3D viewer to inspect your design by panning, zooming, and toggling layer visibility. open the 3d viewer
+    guide to the viewer, what each part does, the short cut keys.
+    - isolate a cell, where are the mosfets
+
+
+3. **Manual Action Check**:
+
+    - Usually, on each commit, the workflow runs automatically, However in a scenario that you changed WOKWI design itself and now you want to check the action result, Manual action workflow is to be opted. 
```

Comment:
WOKWI -> Wokwi
no need to capitalise 'However' as it's not starting a new sentence.

Change the whole sentence to:

 Usually when you commit, the GitHub action will be automatically started. However, if you change the Wokwi design and you want to trigger the actions manually, you can use the 'run workflow' button.